tmdbsimple/base.py

```diff
@@ -102,6 +102,8 @@ def _set_attrs_to_values(self, response={}):
         """
         if isinstance(response, dict):
             for key in response.keys():
-                if not hasattr(self, key):
+                if not hasattr(self, key) or key in {'results', 'page', 
```

Comment:
Or maybe we can check if the caller is an instance of `Search`?  
Something like this:  
```python
if not hasattr(self, key) or isinstance(self, Search):
```  
I am pretty sure this is wrong. I don't think the `Search` object is referred to like this.